src/layout/List/ListTheme.ts

```diff
@@ -37,7 +37,7 @@ const baseTheme: ListTheme = {
     disableGutters: 'pl-0 pr-0',
     dense: {
       base: 'p-1',
-      content: 'text-xs',
+      content: '',
```

Comment:
Should we make this small instead?

Reply:
It's small from the text-sm on the non-dense content. Now dense and not dense lists are the same font size but different padding